### DISCUSSION TITLE
What are the system requirements for ANTs

### DISCUSSION THREAD

**Discussion by Snyper1701** [ANTsX/ANTs#1792] [Q&A]
I want to install ANTs on 2 different machines. I searched the repository for any hint to the required sytem specs but couldn't find any. 
Before I attempt to install ANTs I need to know the following:

Processor (does it matter if it is an Intel or AMD)
RAM required
Graphics card required (aka is OpenGL neccessary)
Python version (if needed)

For explaination: We have two computing machines. Both run under Linux (Ubuntu 22 LTS and Debian 11). One of them is vitualized and does not provide hardware graphics accelaration, while the other does but does not have as much RAM.

Thank you in advance

**Comment by gdevenyi** [comment]
Requirements are entirely dependent on the data you are processing.

I have some (old) benchmarks on the scaling of ANTs for the core registration tool here:
https://github.com/gdevenyi/antsRegistration-benchmarking/blob/master/Rplots.pdf
